### PR TITLE
Readds BZ Cannisters for Xenobiology

### PR DESCRIPTION
**Pull request by YakumoChen**
:cl: YakumoChen
tweak: Nanotrasen has once again shipped BZ to its nearby stations. Check your local Xenobiology lab for your shipment.
tweak: Nanotrasen would like to remind you that BZ is meant for keeping slimes in stasis. It is not for human use and may lead to hallucinations. Always practice workplace safety when handling dangerous gasses!
/:cl:

Hopefully Works This Time Edition
[why]: BZ has exactly one beneficial use. When it was created, it replaced Carbon Dioxide as the gas of choice for putting slimes into stasis, which pauses movement and hunger. Unfortunately, it was soon upended by Goof's Meme Freon and the canister you would find in Toxins was removed when Freon was made.
I have added a canister directly to the Xenobio labs on each map (except Omega) to encourage Xenobiologists to use them for their work. Currently, two maps (Meta and Pubby) also conveniently have a gas distribution system for their large chambers and the BZ Canister has been placed on top their respective connector ports, to again, encourage use. Box and Delta Xenoscientists will just have to make do and put on a gas mask to spread gas in their large containment.

And yes, of course, you could just open the canister in the hall and give everyone hallucinations. Nobody's stopping you from doing that either.

### CODE REVIEW

**Comment by YakumoChen** [comment]
ha ha it works this time cant pen the chen friends

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -37604,6 +37603,9 @@
 	dir = 1
 	},
 /obj/structure/table/reinforced,
+/obj/item/device/slime_scanner{
+	step_x = 0
```
Comment (81Denton) [inline _maps/map_files/BoxStation/BoxStation.dmm]: dv

Comment (YakumoChen) [reply _maps/map_files/BoxStation/BoxStation.dmm]: I think I have to completely redo it in the map because I can't hand edit the file

**Comment by PKPenguin321** [comment]
Freeze

**Comment by YakumoChen** [comment]
Aw heck do I not get my one free?

**Comment by PKPenguin321** [comment]
I think the window closed for that when the thaw PR closed

**Comment by YakumoChen** [comment]
It's small and this is my first actual time coding but I guess I will leave it up to the big guns :tm: to decide

**Comment by DaxDupont** [comment]
Don't add bz back, it's only gonna be used for grief. Order it from cargo and don't have this available on round start.

**Comment by Supermichael777** [comment]
>cargo order
>ever getting used for anything but cargo griff

**Comment by davethwave** [comment]
so instead of ONE possible canister of bz you want who knows how many that money could buy?

**Comment by YakumoChen** [comment]
BZ has exactly one explicitly intended use, via Xenobio, that has never ever seen use because you can't get BZ on station except through cargo and I dare you to find a xenobiologist that has actually gone through the effort for a niche use they probably also doesn't know exists.

**Comment by Kromgar** [comment]
Does BZ put slime people into stasis?

**Comment by Cyberboss** [comment]
Try again after the freeze

**Comment by YakumoChen** [comment]
Slimes into stasis isn't something I would consider worthy of discovery. At best, it's a convenience for Xenobiology, and not something players are going to want to actively seek out if they know about it, and certainly probably not going to be worth the trouble of ordering a canister from cargo ever since even its use as a grief gas is highly niche. I might, in fact, prefer its removal and reinstating CO2 for slime stasis, but alas, I'm not a coder.

**Pull request closed without merging**